src/amo/components/AddonDetail.js

```diff
@@ -28,13 +58,13 @@ export class AddonDetail extends React.Component {
             <LikeButton />
           </div>
           <div className="title">
-            <h1>Placeholder Add-on Title
-              <span className="author">by <a href="#">AwesomeAddons</a></span></h1>
-            <InstallButton slug="placeholder" />
+            <h1>
+              {addon.name} <span className="author">{i18n.gettext('by')} {authorList}</span>
```

Comment:
Good catch!

If we only translate `by` the downside would be if a locale needed to change the word order for some reason it couldn't.

In the past we've done things like this to be able to expose the full context for translation allthought for disco pane the string was built on the server.

```jsx
{i18n.sprintf(i18n.gettext('%(addonName)s %(startSpan)sby %(authorList)s%(endSpan)s'), { addonName, authorList, startSpan, endSpan })}
``` 

If passed through domurify if the tags are reversed via the translation you will still get a sane output albeit the markup won't be wrapping the things it should, but you will at least get valid HTML.

e.g:

```html
</span>foo<span>
```

becomes:

```html
foo<span></span>
```